src/lib.rs

```diff
@@ -1,7 +1,6 @@
 #![doc = include_str!("../README.md")]
-
 #![forbid(unsafe_code, unused_must_use)]
-#![forbid(
+#![warn(
```

Comment:
Hmm... Are you sure about that?
I would prefer a clear forbid if possible. I mean I started this project for myself at the beginning, but if some people decide to contribute I would rather prefer to make documentation mandatory.